contracts/deploy/01-outbox/01-arb-to-gnosis-outbox.ts

```diff
@@ -36,17 +36,17 @@ const paramsByChainId = {
     sequencerLimit: 86400, // 24 hours
   },
   HARDHAT: {
-    deposit: parseEther("5"), // 120 xDAI budget for timeout
+    deposit: parseEther("10"), // 120 xDAI budget for timeout
     // Average happy path wait time is 22.5 mins, assume no censorship
     epochPeriod: 600, // 10 min
-    challengePeriod: 600, // 10 min (assume no sequencer backdating)
-    numEpochTimeout: 24, // 6 hours
+    minChallengePeriod: 600, // 10 min (assume no sequencer backdating)
+    numEpochTimeout: 21600, // 6 hours
```

Comment:
Looks like the value changed but the comment is still the same. Is the new value the corrected number for 6 hours?

Reply:
previous value was wrong , unit being used here is second so 21600 second equals 6 hrs